ext/dcos-installer/dcos_installer/action_lib/configure.py

```diff
@@ -74,15 +85,19 @@ def cleanup_and_exit():
 
     # Copy out the files, rolling back if it fails
     try:
-        subprocess.check_output(['mkdir', '-p', '/genconf/serve/bootstrap/'])
+        subprocess.check_output(['mkdir', '-p', dest_dir])
 
         # Copy across
         for filename in copy_set:
-            subprocess.check_output(['cp', container_cache_dir + filename, dest_dir + filename])
+            subprocess.check_output(['cp', os.path.join(artifact_dir, filename), os.path.join(dest_dir, filename)])
     except subprocess.CalledProcessError as ex:
         log.error("Copy failed: %s\nOutput:\n%s", ex.cmd, ex.output)
         log.error("Removing partial artifacts")
         cleanup_and_exit()
     except KeyboardInterrupt:
         log.error("Copy out of installer interrupted. Removing partial files.")
         cleanup_and_exit()
+
+
+def fetch_packages(artifact_dir, dest_dir):
```

Comment:
Why not just inline this? It's a single line, the arguments are always passed as constants.